README.md

```diff
@@ -1,11 +1,19 @@
 # Migrate with doctrine migrations
 
+This extension uses `doctrine/migrations` to migrate the database tables.
+
 __It's strongly recommend to use the "typo3" CLI binary!__
 
 To get the status of your migrations you can run:
 
     <path-to-bin>/typo3 migrations:status
 
+To execute one pending migrations you can run:
+
+    <path-to-bin>/typo3 migrations:migrate <pending-version-id>
+
+(Hint: To roll back a migration pick the version before the one you wanna rollback!)
```

Comment:
That would possibly roll back more than this one migration, right?

There is also `migrations:execute` which could be used to roll back a single migration (with the `--down` parameter).